frappe/utils/data.py

```diff
@@ -1707,6 +1707,8 @@ def get_url_to_report_with_filters(name, filters, report_type=None, doctype=None
 	"<=": operator.le,
 	"not None": lambda a, b: a is not None,
 	"None": lambda a, b: a is None,
+	"like": lambda a, b: operator.contains(a.strip("%"), b.strip("%")),
+	"not like": lambda a, b: not operator.contains(a.strip("%"), b.strip("%")),
```

Comment:
@surajshetty3416 just a question on design

Should we support SQL like `like` filtering where prefix/suffix match is supported?